### ISSUE TITLE
'React/RCTConvert.h' file not found

### ISSUE THREAD

**Issue by sujitpk-perennial** [react-native-community/react-native-svg#748]
Hello,

  I am getting error as `'React/RCTConvert.h' file not found` after adding firebase dependency in pod file and doing `pod update`.
   
 I am getting above error in class RNSVGBrush

>   react-native-svg - ^6.3.1
>   react-native - ^0.50.1
>   react - 16.0.0-alpha.12
> 
> 

  I am stuck here from last few hours.

**Comment by svegan** [comment]
Have same issue. Have you foung a solution?

**Comment by SnowDang** [comment]
I have the same issue.

**Comment by ighotouch** [comment]
Hello is there any solution to this?

**Comment by msand** [comment]
Seems like a linking issue. Hard to give any suggestions with the current amount of information, as I haven't seen/resolved the issue myself, and am not even using cocoapods for anything atm. Could you make a minimal reproduction? Start with initialising a new react-native project and git repository, configure cocoapods, make sure everything works, commit. Add react-native-svg and firebase as dependencies, check if it still fails. Then commit, push to some public git repo and report back with a link to the git repo.

**Comment by saggiyogesh** [comment]
Fixed by making the following changes to `Podfile`
```

pod 'React', :path => '../node_modules/react-native', :subspecs => [
    'Core',
    'CxxBridge',
    'DevSupport',
    # the following ones are the ones taken from "Libraries" in Xcode:
    'RCTAnimation',
    'RCTActionSheet',
    'RCTBlob',
    'RCTGeolocation',
    'RCTImage',
    'RCTLinkingIOS',
    'RCTNetwork',
    'RCTSettings',
    'RCTText',
    'RCTVibration',
    'RCTWebSocket'
  ]
  
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/GLog.podspec'
  
pod 'RNSVG', :path => '../node_modules/react-native-svg'

post_install do |installer_representation|
    installer_representation.pods_project.targets.each do |target|
        if target.name == "React"
            target.remove_from_project
        end
    end
end
```

In my case `react-native-svg` installed React pod which's already there in node_modules, hence creating conflict.
So declared React pod, referencing to pod exists in node_modules/react-native

**Comment by jaygould** [comment]
@saggiyogesh I have tried this solution and I think I'm getting one step closer, however  I am getting an error: 

ld: 63 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Do you have any idea what this could be?

![screen shot 2018-09-28 at 17 34 51](https://user-images.githubusercontent.com/13678558/46221383-d693c100-c344-11e8-815e-607861c22f99.png)

**Comment by saggiyogesh** [comment]
I think the error is with `react-native-video`. Try to clear everything & then reinstall. 

- rm -rf node_modules/
- npm / yarn install
- pod update 

Hope this works.

**Comment by Zurab1996** [comment]
same @jaygould

**Comment by jaygould** [comment]
@Zurab I ended up spending a few hours and running in to issue after issue, so decided to download a fresh install of RN 0.57 and copying over my git folder, src, and all settings etc, re installing via NPM and re-linking. All works like a charm now and runs a bit faster.

**Comment by msand** [comment]
This is probably an issue with react-native & cocoapods, not this library. Closing...

**Comment by nabilfreeman** [comment]
Hi there, I fixed this by removing `RNSVG` completely from my `Podfile` and then manually linking `RNSVG` into the "Libraries" group in Xcode.

The cause was as mentioned above, `React` is declared as a dependency in this Pod and we already have it linked outside Cocoapods. In fact we are almost 100% Pod-free!

NB: I had to add it to "Link Binary With Libraries" for **all** the targets in my app.

I think the `podspec` for this library has been written badly so it's best to avoid it.
